README.md

```diff
@@ -512,21 +562,6 @@ The following settings are available per profile:
   Elasticsearch in stacks managed by elastic-package. It is recommended to use
   an absolute path, out of the `.elastic-package` directory.
 
-## Development
```

Comment:
This section was duplicated. Merged into the other Development section in line 39